extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/HttpConfiguration.java

```diff
@@ -197,6 +198,18 @@
     @ConfigItem
     public Map<String, SameSiteCookieConfig> sameSiteCookie;
 
+    /**
+     * Configuration that allows setting forwarded host header to be considered.
+     */
+    @ConfigItem(defaultValue = "x_forwarded_host")
+    public ForwardedHostHeader forwardedHostHeader;
+
+    /**
+     * Configuration that allows setting forwarded prefix header to be considered.
+     */
+    @ConfigItem(defaultValue = "x_forwarded_prefix")
+    public ForwardedPrefixHeader forwardedPrefixHeader;
```

Comment:
I think this should just be a string, and it should default to being empty (i.e. this needs to be explicitly enabled). 

These headers have serious security considerations, so they need to be specifically enabled for the environment the user is actually in, we can't just enable them by default or attackers may be able to spoof information.

Reply:
Both properties are Optional<String> now.